spec/index_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+
+describe 'Index Controller' do
+
+  it 'Should get the / route' do
+    get '/'
+    expect(last_response).to be_ok
+    expect(last_response.body).to include('Sealions')
+  end
+
+  #CREATE
+  it 'Should process a post request' do
+    post '/sealions'
+    expect(last_response).to be_redirect
+    follow_redirect!
+    expect(last_response).to be_ok
+    expect(last_response.body).to include('Sealions')
+  end
+
+  #READ
+  it 'Should process a read request on id 3' do
+    get '/sealions/:ID'
+    expect(last_response).to be_ok
+    expect(last_response.body).to include("Sealion")
+  end
+
+  #UPDATE
+  it 'Should process a put request on id 3' do
+    put '/sealions/:ID'
```

Comment:
You want to put '/sealions/22' or something. Right now, I think that `:ID` is not actually being extracted by the route. The `put` route is literally matching the exact string '/sealions/:ID', and that's the route your specs are hitting.